raiden-ts/src/transport/epics/init.ts

```diff
@@ -209,7 +192,8 @@ function fetchSortedMatrixServers$(matrixServerLookup: string, httpTimeout: numb
       return response.text();
     }),
     timeout(httpTimeout),
-    mergeMap((text) => yamlListToArray(text)),
+    map((text) => decode(MatrixServerInfo, JSON.parse(text), new RaidenError(ErrorCodes.TRNS_NO_MATRIX_SERVERS))),
```

Comment:
@andrevmatos Any hints on how to handle exceptions in `JSON.parse` here? And should I add a new error message for this?

Reply:
I'd suggest not calling `JSON.parse` directly. Instead, `return response.json()` above, and decode the resolved `json` object. I'd suggest also not passing an explicit error here, let the decode error flow (it can be more informative), and then catch it later, I think it's already handled on the function which calls `fetchSortedMatrixServers$`

Reply:
If you need to parse the text by yourself, use `jsonParse` utility